src/components/custom/filter/filter-item-checkbox/types.ts

```diff
@@ -0,0 +1,13 @@
+import { ICustomFieldJsonSchema } from "../../../frontend-engine";
+export interface IFilterItemCheckboxSchema<V = undefined> extends ICustomFieldJsonSchema<"filter-item-checkbox", V> {
```

Comment:
sorry just spotted this, use `filter-checkbox` instead of `filter-item-checkbox`